Automattic/jetpack
# VideoPress: Show video quick action tooltip on keyboad navigation

Depends on https://github.com/Automattic/jetpack/pull/27378

Fixes #27332

#### Changes proposed in this Pull Request:

* Shows tooltip on focus on video quick action

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Go to the VideoPress dashboard with at least one video
* Using `tab` to navigate, go to a quick action
* Check that the tooltip is shown

https://user-images.githubusercontent.com/8486249/201430071-1f7cef27-f627-4e5f-a256-ba31338b23ad.mp4
